src/util/Bundles.js

```diff
@@ -155,12 +155,12 @@ function fetchJson(bundle, language, assets) {
     .then(txt => JSON.parse(txt));
 }
 
-function getBundles(language, assets) {
+function getBundles(language, assets, supportedLanguages) {
   // Two special cases:
   // 1. English is already bundled with the widget
   // 2. If the language is not in our config file, it means that they've
   //    probably defined it on their own.
-  if (language === 'en' || !_.contains(config.supportedLanguages, language)) {
+  if (language === 'en' || !_.contains(supportedLanguages || config.supportedLanguages, language)) {
```

Comment:
Looks like `config.supportedLanguages` is sub-set of `supportedLanguages`.
Do we have to check both?